packages/core/src/line-chart.ts

```diff
@@ -35,6 +35,15 @@ export class LineChart extends BaseAxisChart {
 		}));
 	}
 
+	getCircleRadius() {
+		return this.options.points.radius || Configuration.charts.pointCircles.radius;
```

Comment:
Why different naming for the same thing in options and configuration? points vs pointCircles?